src/app/shared/util/date.util.spec.ts

```diff
@@ -1,13 +1,14 @@
 import { isMoreThanXDays, beginningOfTodayUTC, endOfTodayUTC,
-  beginningOfThisWeekUTC, beginningOfTwoWeeksUTC, beginningOfThisMonthUTC, beginningOfThreeMonthsUTC, beginningOfThisYearUTC,
-  beginningOfYesterdayUTC, endOfYesterdayUTC, beginningOfLastWeekUTC, endOfLastWeekUTC,
-  beginningOfPriorTwoWeeksUTC, endOfPriorTwoWeeksUTC, beginningOfLastMonthUTC, endOfLastMonthUTC,
-  beginningOfPriorThreeMonthsUTC, endOfPriorThreeMonthsUTC, beginningOfLastYearUTC, endOfLastYearUTC,
+  beginningOfThisWeekUTC, beginningOfLastWeekUTC, endOfLastWeekUTC, beginningOfLast7DaysUTC,
+  beginningOfThisWeekPlus7DaysUTC,
+  beginningOfThisMonthUTC, beginningOfLastMonthUTC, endOfLastMonthUTC, beginningOfLast28DaysUTC, beginningOfLast30DaysUTC,
+  beginningOfThisMonthPlusTwoMonthsUTC, beginningOfLast90DaysUTC,
+  beginningOfThisYearUTC, beginningOfLast365DaysUTC,
   getBeginEndDateFromStandardRange, getStandardRangeForBeginEndDate, getMillisecondsOfInterval,
   roundDateToBeginOfInterval, roundDateToEndOfInterval, getAmountOfIntervals,
   UTCDateFormat, dayOfWeekDateFormat, dayMonthDateFormat, monthDateYearFormat, monthYearFormat, hourlyDateFormat,
-  TODAY, YESTERDAY, THIS_WEEK, LAST_WEEK, TWO_WEEKS, PRIOR_TWO_WEEKS, THIS_MONTH, LAST_MONTH,
-  THREE_MONTHS, PRIOR_THREE_MONTHS, THIS_YEAR, LAST_YEAR } from './date.util';
+  THIS_WEEK, LAST_WEEK, LAST_7_DAYS, THIS_WEEK_PLUS_7_DAYS, THIS_MONTH, LAST_MONTH, LAST_28_DAYS, LAST_30_DAYS,
+  THIS_MONTH_PLUS_2_MONTHS, LAST_90_DAYS, THIS_YEAR, LAST_365_DAYS, OTHER } from './date.util';
```

Comment:
Could also just import * here.

Reply:
Not using import * is actually killing me but I guess I kinda thought we were anti import * and sided more with being specific with imports, no unused imports. I'd prefer using barrel files and doing import * in most cases.

Reply:
Yeah, if it's just a few things (or angular/ngrx core components), I usually side with specific imports.  But yeah, these huge imports are starting to get to me.  Or Todd Motto is subliminally influencing me.